### ISSUE TITLE
computeCOPtrajectory and sensitivity 

### ISSUE THREAD

**Issue by eleonora-mont** [GallVp/footPress#3]
Hello @GallVp 

I am dealing with frames of pressure which were collected in an uncontrolled setting, hence foot placement on the pressure platform was highly variable. Therefore, when I import my frames in matlab, I pad them in a grid of 36x25 to ensure standard matrices dimensions among frame, but this doesn't change the fact that frames of each step across the data set are oriented differently with respect to the matrix reference system. As a result, when I compute COP trajectory with the computeCOPtrajectory function, I get that the resulting starting point of the medio-lateral and anterior-posterior COP trajectories are highly different across the different steps. I was wondering if this happened to you before and how you dealt with it in that case? 

Thanks,

Eleonora

**Comment by GallVp** [comment]
Hi @eleonora-mont 

No I did not face any such issues as we used pressure sensitive shoes in which there was minimal movement between the foot and the shoe. Thus, the frame of reference of the foot and of the pressure sensor array were always aligned.

For your case, I can't make a definitive suggestion without seeing the data. Using reference frame translations and rotations to align landmarks such as heel, first and fifth metatarsal with lower centre, top left and top right (opposite for left foot) may help resolve your issue.
